x-pack/plugins/observability/public/utils/is_alert_details_enabled.ts

```diff
@@ -15,6 +15,7 @@ export const isAlertDetailsEnabledPerApp = (
   alert: TopAlert | null,
   config: ConfigSchema | null
 ): boolean => {
+  console.log(alert, config);
```

Comment:
This is the only file change that we own, so after removing it there is no need for review from AO

Reply:
thanks! missed this